python/lib/scheduler/dmod/scheduler/scheduler.py

```diff
@@ -364,6 +551,17 @@ def start_job(self, job: 'Job') -> Tuple[bool, tuple]:
 
         service_per_allocation = []
 
+        # Get the Docker Secrets for object store data access
+        # TODO (later): might need to expand to use different users for different situations (create JobType?)
+        secrets = [self.docker_client.secrets.get('object_store_exec_user_name'),
+                   self.docker_client.secrets.get('object_store_exec_user_passwd')]
+
+        # Add this to env variables, depending on whether this is a single node (Desktop) or multi-node setup
+        if len(self.docker_client.nodes.list()) == 1:
+            env_vars = {'MINIO_DEPLOYMENT': 'standalone'}
```

Comment:
Once again am wondering why the proxy isn't used even on a single host???

Reply:
I don't think I intended to have this for anything other than the minio setup, and searching the branch now, I don't find any usage of this string elsewhere.  I'll take it out also.